src/components/Header/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
+import Head from 'next/head';
+import Image from 'next/image';
+import styles from './NavBar.module.css';
+
+export default function NavBar() {
+    return (
+        <div className={styles.container}>
+            <Image
+                src="/vercel.svg"
+                alt="Vercel Logo"
+                className={styles.vercelLogo}
+                width={100}
+                height={24}
+                priority
+              />
+            
+            <form>
```

Comment:
I would move this into its own component and import it here.

Reply:
I can definitely move the form into it's own component, thanks @jjleigh

Reply:
Remember to create a ticket for this task!